src/features/workflow/state/workflowState.ts

```diff
@@ -63,20 +77,20 @@ const initialState: WorkflowState = {
   runMode: 'apply',
   workflow: {
     id: '',
-    datasetID: 'fake_id',
-    runCount: 0,
+    initID: 'new_dataset',
```

Comment:
does `initID` need 'new_dataset'? If it can be an empty string, it would probably be better in the long term, I can see if accidentally intro-ing a bug at some point.